### PR TITLE
Double backbone model fix

### PR DESCRIPTION
**Pull request by devmanhinton**
@yaymukund Fix the spec and bump the version!

### CODE REVIEW

File: test/non_backbone_integration.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
+require('./support/helper');
+
+var path = require("path")
+  , expect = require("expect.js")
+  , fs = require("fs")
+  , generateTemplate = require("./support/generate_template")
+  , Backbone = require('Backbone')
```
Comment (yaymukund) [inline test/non_backbone_integration.js]: This should be lowercase.

Comment (devmanhinton) [inline test/non_backbone_integration.js]: Yeah, swear I did that.

**Comment by devmanhinton** [comment]
@yaymukund Can you merge?

File: test/non_backbone_integration.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+describe("With two copies of Backbone", function(){
+  before(function(){
+    delete require.cache[require.resolve('backbone')];
+    BackboneClone = require('backbone');// Internally we use !(model instanceof Backbone.Model || ... instanceof Backbone.Collection)
+                                        // If the Backbone for EndDash is different then the clients, we want to handle this case.
+                                        // See lib/reactions/model.js for fix.
+  })
+  after(function(){
+    delete require.cache[require.resolve('backbone')]
```
Comment (yaymukund) [inline test/non_backbone_integration.js]: These two lines should be terminated with a semicolon..

**Pull request merged**